Python/hamt.c

```diff
@@ -1176,7 +1176,7 @@ hamt_node_bitmap_dump(PyHamtNode_Bitmap *node,
         }
 
         if (key_or_null == NULL) {
-            if (PyUnicodeWriter_WriteUTF8(writer, "NULL:\n", -1) < 0) {
+            if (PyUnicodeWriter_WriteASCII(writer, "NULL:\n", -1) < 0) {
```

Comment:
```suggestion
            if (PyUnicodeWriter_WriteASCII(writer, "NULL:\n", 6) < 0) {
```

I think it's 6 here right?

Reply:
Sorry, I don't understand your question. I replaced -1 with 6 on purpose.

Reply:
My question was: did I count it right? in your PR, it's -1.

Reply:
Oh sorry, I hesitated to write 6 but I wasn't sure :-D I was confused by the GitHub UI. I applied your change.